src/config/schema.ts

```diff
@@ -87,13 +88,15 @@ export const chainsSchema = z.record(optionalChainSchema).transform((chains, ctx
     Object.entries(chains).map(([chainId, chain]) => {
       const { contracts } = chain;
       const parsedContracts = contractsSchema.safeParse({
+        ...contracts,
         Api3ServerV1: contracts?.Api3ServerV1 ?? references.Api3ServerV1[chainId],
       });
       if (!parsedContracts.success) {
         ctx.addIssue({
           code: 'custom',
           message: 'Invalid contract addresses',
-          path: ['chains', chainId, 'contracts'],
+          // Show at least the first error.
+          path: [chainId, 'contracts', ...parsedContracts.error.errors[0]!.path],
```

Comment:
I realized `chains` should not be part of the message. Also it would be nice if we showed what address was wrong and we already have that information.